src/components/Votes.jsx

```diff
@@ -0,0 +1,46 @@
+import { useState } from "react";
+import { getUpdateVotes } from "../axios/apiQueries";
+
+const Votes = ({ review_id, votes }) => {
+  const [userVote, setUserVote] = useState(0);
+  const [updatedVotes, setUpdatedVotes] = useState(votes);
+
+  const onVoteUp = () => {
+    setUserVote(1);
+    setUpdatedVotes(updatedVotes);
```

Comment:
`setUpdatedVotes(updatedVotes)`

^ Why are you setting the updateVotes to be what it already is? This seems unnecessary?